app/views/peoplefinder/people/_day.html.haml

```diff
@@ -1,3 +1,4 @@
 %li{class: ('active' if @person.send(day)), alt: day_name(day)}
-  .visuallyhidden= "Works #{ day_name(day) }" if @person.send(day)
+  .visuallyhidden= "Works on #{ day_name(day) }s" if @person.send(day)
+  .visuallyhidden= "Doesn't work on #{ day_name(day) }s" if not @person.send(day)
```

Comment:
It seems to always produce 2x `div.visuallyhidden` and one of them is empty. I guess the `if` condition only applies to the content of the div, rather to the div itself. I'm not sure how HAML handles it.